packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -306,4 +306,358 @@
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @selected={{true}} @count="12">
+            {{state}}
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Custom content">
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @count="12">
+            <Shw::Placeholder @text="custom content" @width="128" @height="20" @background="#eee" />
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Custom content, selected">
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @selected={{true}} @count="12">
+            <Shw::Placeholder @text="custom content" @width="128" @height="20" @background="#eee" />
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Badge in content">
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}}>
+            {{state}}
+            <Hds::Badge @icon="org" @text="Private" @size="small" />
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Badge in content, selected">
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @selected={{true}}>
+            {{state}}
+            <Hds::Badge @icon="globe" @text="Public" @size="small" @color="highlight" />
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Large content">
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @icon="hexagon" @selected={{true}} @count="12">
+            {{state}}
+            with a longer text string that may wrap since max-width is defined on the container
+            <Hds::Badge @text="badge" @size="small" />
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item @label="Interactive">
+      <Hds::Dropdown @listPosition="left" as |dd|>
+        <dd.ToggleButton @text="Checkmark" @color="secondary" />
+        <dd.Checkmark @count="11">virtualbox</dd.Checkmark>
+        <dd.Checkmark @count="1" @selected={{true}}>vmware</dd.Checkmark>
+        <dd.Checkmark @count="10">docker</dd.Checkmark>
+        <dd.Checkmark @count="0">hyperv</dd.Checkmark>
+      </Hds::Dropdown>
+    </SF.Item>
+  </Shw::Flex>
+
```

Comment:
I would probably add level-1 and level-2 separators (`<Shw::Divider />`, `<Shw::Divider @level={{2}} />`) between the different types examples, to increase readability/scan.
See for example how it's done in the [Pagination](https://hds-showcase.vercel.app/components/pagination) or [Form Base elements](https://hds-showcase.vercel.app/components/form/base-elements) showcases.

Reply:
Added the level 2 dividers – looks much better with them!